archai/common/notebook_helper.py

```diff
@@ -0,0 +1,48 @@
+from typing import Union
```

Comment:
Let's try to keep in every Archai-ownership file:

```
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT license.`
```

Reply:
Done.